graphql/src/adapters/logger_provider/basic.py

```diff
@@ -1,25 +0,0 @@
-import logging
```

Comment:
I did a proper git -mv to rename the file, so I have no idea why it's deleted in the diff.